qutebrowser/utils/utils.py

```diff
@@ -810,6 +811,11 @@ def get_clipboard(selection=False):
         mode = QClipboard.Selection if selection else QClipboard.Clipboard
         data = QApplication.clipboard().text(mode=mode)
 
+    target = "Primary selection" if selection else "Clipboard"
+    if not data.strip():
+        raise cmdexc.CommandError("{} is empty.".format(target))
```

Comment:
Raising a `CommandError` from an utility function seems like a wrong thing to do - I'd prefer if you added a `ClipboardEmptyError` or so, raised that here, and then handled and re-raised that in the command handlers. Right now, this is also unhandled in `miscwidgets.py` where `utils.get_clipboard()` is used too.